.travis.yml

```diff
@@ -41,7 +41,7 @@ matrix:
 
     - dist: trusty
       env:
-        - JOB="3.6, coverage" ENV_FILE="ci/deps/travis-36.yaml" PATTERN="((not slow and not network) or (single and db))" PANDAS_TESTING_MODE="deprecate" COVERAGE=true
+        - JOB="3.6, coverage" ENV_FILE="ci/deps/travis-36-cov.yaml" PATTERN="((not slow and not network) or (single and db))" PANDAS_TESTING_MODE="deprecate" COVERAGE=true
```

Comment:
I made the name of the coverage build more explicit  `travis-36` -> `travis-36-cov`. It had some pins it should not have had, and it's easy to forget that when not being reminded by the name (same for the `travis-36-doc` build).